code/modules/surgery/organs/organ_internal.dm

```diff
@@ -41,6 +41,8 @@
 			M.internal_organs_slot.Remove(slot)
 		if(vital && !special)
 			M.death()
+		if(loc == M)
+			forceMove(get_turf(src))
```

Comment:
Anything that removes organs codewise should already be moving it somewhere. Line 23, for instance.